lib/playdate_web/live/apps_live.ex

```diff
@@ -23,13 +23,18 @@ defmodule PlaydateWeb.LobbyLive.Index do
     current_active_apps =
       Apps.AppInstance
       |> Ash.Query.load([:app])
-      |> Ash.Query.filter(state in ["waiting-for-users", "ready-to-start"])
       |> Apps.read!()
+      |> Stream.map(fn app_instance ->
```

Comment:
```suggestion
      |> Map.new(fn app_instance ->
```

[Map.new/2](https://hexdocs.pm/elixir/1.14.0/Map.html#new/2) can replace most usages of the old `Enum.into(%{})` idiom.